src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,52 @@
-import React from 'react';
+/* eslint-disable @typescript-eslint/quotes */ import React from "react";
 
-import './MovieCard.scss';
+import "./MovieCard.scss";
+import { Movie } from "../../types/Movie";
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+type PropsType = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<PropsType> = (props: PropsType) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+    imdbId,
+  } = props.movie;
+
+  return (
+    <div key={imdbId} className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img data-cy="MovieImage" src={imgUrl} alt="Film logo" />
```

Comment:
```suggestion
          <img data-cy="MovieImage" src={imgUrl} alt="Movie logo" />
```
lets do it this way to be consistent :)